backend/src/routes/webhooks.js

```
@@ -0,0 +1,494 @@
+      console.error('Stripe webhook signature verification failed:', err.message);
+      return res.status(400).send(`Webhook Error: ${err.message}`);
+    }
+
+    const Payment = createPaymentModel(sequelize());
+    const Transaction = createTransactionModel(sequelize());
+    const User = createUserModel(sequelize());
+    const Notification = createNotificationModel(sequelize());
+
+    switch (event.type) {
+      case 'payment_intent.succeeded':
+        const paymentIntent = event.data.object;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)